docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.traditional-pairings.scss

```diff
@@ -0,0 +1,30 @@
+// TRADITIONAL
+
+// Set the default color pairing value
+// The default color pairing also serves as the fallback color
+// pairing for browsers that do not support CSS custom properties
+$hb-traditional-default: 'default';
+
+// Each color pairing has a palette of color swatches
+$ht-traditional-pairings: (
```

Comment:
If these are placeholder pairings (not finalized) then we should probably comment the code to say this. This way, we're less likely to have people trying to test their work on these pairings.